src/doc/data_into_visit/XdmfFormat.rst

```diff
@@ -3,4 +3,298 @@
+
+Here is the structure of a basic Xdmf file. ::
+
+    <?xml version="1.0" ?>
+    <!DOCTYPE Xdmf SYSTEM "Xdmf.dtd" []>
+    <Xdmf Version="3.0">
+     <Domain>
+       <Grid ... >
+         <Topology ... />
+         <Geometry ... >
+           <DataItem ... >
+             ...
+           </DataItem>
+         </Geometry>
+         <Attribute ... >
+           <DataItem ... >
+             ...
+           </DataItem>
+         </Attribute>
+       </Grid>
+     </Domain>
+    </Xdmf>
+
+DataItem
+~~~~~~~~
+
+The `DataItem` is used to store embedded data or external binary data.
+
+The following is an example of a `DataItem` that embeds the data directly in the Xdmf file. ::
+
+    <DataItem Format="XML" NumberType="Float" Precision="4" Dimensions="3">
+        1.0 2.0 3.0
+    </DataItem>
+
+The following is an example of a `DataItem` that references an array in an HDF5 file. ::
+
+    <DataItem Format="HDF" NumberType="Float" Precision="4" Dimensions="3">
+        output.h5:/values
+    </DataItem>
+
+The following is an example of a `DataItem` that references an array in a binary file. ::
+
+    <DataItem Format="Binary" NumberType="Float" Precision="4" Endian="Little" Seek="0" Dimensions="3">
+        output.bin
+    </DataItem>
+
+The valid values for `Format` are:
+
+* XML
+* HDF
+* Binary
+
+The valid values for `NumberType` are:
+
+* Float
+* Int
+* UInt
+* Char
+* UChar
+
+The valid values for `Precision` are:
+
+* 1 - Char or UChar
+* 2 - Int or UInt
+* 4
+* 8
+
+`Dimensions` consists of one to three values representing a 1D, 2D or 3D array.
+
+The valid values for `Endian` are:
+
+* Native
+* Big
+* Little
+
+`Seek` is an byte offset into a binary file.
+
+Topology
+~~~~~~~~
+
+The following mesh topologies are supported.
+
+* Structured
+
+  * 2DSMesh - Curvilinear
+  * 2DRectMesh - Rectilinear
+  * 2DCoRectMesh - Regular
+  * 3DSMesh - Curvilinear
+  * 3DRectMesh - Rectilinear
+  * 3DCoRectMesh - Regular
+
+* Unstructured
+
+  * Linear
+
+    * Polyvertex - a group of unconnected points
+    * Polyline - a group of line segments
+    * Polygon
+    * Triangle
+    * Quadrilateral
+    * Tetrahedron
+    * Pyramid
+    * Wedge
+    * Hexahedron
+
+  * Quadratic
+
+    * Edge_3 - Quadratic line with 3 nodes
+    * Tri_6
+    * Quad_8
+    * Tet_10
+    * Pyramid_13
+    * Wedge_15
+    * Hex_20
+
+  * Arbitrary
+
+    * 1 - POLYVERTEX
+    * 2 - POLYLINE
+    * 3 - POLYGON
+    * 4 - TRIANGLE
+    * 5 - QUADRILATERAL
+    * 6 - TETRAHEDRON
+    * 7 - PYRAMID
+    * 8 - WEDGE
+    * 9 - HEXAHEDRON
+    * 16 - POLYHEDRON
+    * 34 - EDGE_3
+    * 35 - QUADRILATERAL_9
+    * 36 - TRIANGLE_6
+    * 37 - QUADRILATERAL_8
+    * 38 - TETRAHEDRON_10
+    * 39 - PYRAMID_13
+    * 40 - WEDGE_15
+    * 41 - WEDGE_18
+    * 48 - HEXAHEDRON_20
+    * 49 - HEXAHEDRON_24
+    * 50 - HEXAHEDRON_27
+
+Geometry
+~~~~~~~~
+
+The following mesh geometries are supported.
+
+* XYZ - Interlaced locations
+* XY - Z is set to 0.0
```

Comment:
When this appears in the generated docs, it almost looks like the option is "XY-Z'
If you make the second column line up, it should make the option clearer.